common/changes/contextualmenu-noitems_2017-01-05-01-07.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Made it so that the contextualmenu will not display if no items are given",
```

Comment:
Fix the comment so that the changelog looks nice.

ContextualMenu: if no items are provided, it will render `null`.